public/components/enterprise/package.go

```diff
@@ -0,0 +1,13 @@
+// Copyright 2024 Redpanda Data, Inc.
+//
+// Licensed as a Redpanda Enterprise file under the Redpanda Community
+// License (the "License"); you may not use this file except in compliance with
+// the License. You may obtain a copy of the License at
+//
+// https://github.com/redpanda-data/connect/blob/main/licenses/rcl.md
+package enterprise
+
+import (
+	// Bring in the internal plugin definitions.
+	_ "github.com/redpanda-data/connect/v4/internal/impl/kafka/enterprise"
+)
```

Comment:
Need to think through this a bit more... Kind of struggling to understand which license headers should go where. Should the following be Apache or RCL? They import RCL components...

- `public/components/all/package.go` 
- `public/bundle/enterprise/package.go`
- `public/components/ollama/package.go`
- `public/components/openai/package.go`
- `public/components/snowflake/package.go`
- `public/components/splunk/package.go`
- ... Any others?

Reply:
I think we kind of figured this out in that this should be RCL right?

Reply:
Yeah any source files within a package that imports an RCL package should be RCL. I think legally it doesn't make any difference but it probably makes life easier for users digging through code and determining what they can and can't access for free.